stats/lib/platform-stats-fetchers.js

```diff
@@ -43,6 +43,22 @@ export const fetchDailyStationAcceptedMeasurementCount = async (pgPool, filter)
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchTopMeasurementStations = async (pgPool, filter) => {
+  const { rows } = await pgPool.query(`
+    SELECT station_id, SUM(accepted_measurement_count) as total_accepted_measurement_count
+    FROM daily_stations
+    WHERE day >= $1 AND day <= $2
+    GROUP BY station_id
+    ORDER BY total_accepted_measurement_count DESC
+    LIMIT 100
```

Comment:
Do we want a limit?

Reply:
Yes, please! There are 7497754 unique station ids in our prod DB now.

```
spark_evaluate=# SELECT COUNT(station_id) FROM daily_stations;
  count  
---------
 9230806
(1 row)

spark_evaluate=# SELECT COUNT(DISTINCT station_id) FROM daily_stations;
  count  
---------
 7497754
(1 row)
```

I think the hard-coded limit 100 is good enough for now. We can increase it in the future or make it configurable via the query string (e.g. `?limit=1000`).

Reply:
There were 3,408 active participants yesterday. I think returning ~3.5k items is acceptable.